### PR TITLE
Add PocketBook 614W (Basic 3) definition

### PR DESCRIPTION
**Pull request by roshavagarga**
Cf. <#6311>.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6312)

### CODE REVIEW

File: frontend/device/pocketbook/device.lua [hunk 1]
```diff
@@ -232,6 +232,16 @@ local PocketBook613 = PocketBook:new{
...
 
+-- PocketBook 614W Basic
+local PocketBook614W = PocketBook:new{ 
+    model = "PB614W",
+    display_dpi = 166,
+    isTouchDevice = no,
+    hasWifiToggle = no,
+    hasFrontlight = no,
+    hasDPad = yes,
+    hasFewKeys = yes, }
```
Comment (Frenzie) [inline frontend/device/pocketbook/device.lua:243]:
```suggestion
    hasFewKeys = yes,
}
```

**Comment by roshavagarga** [comment]
Just a small overview of those changes:
1. I did not try dpi of 167, but it might work, just saw 166 in device specs, either one may be correct.
2. I'm not sure what hasWifiToggle means, I guessed that it meant having a button to enable/disable WiFi, so I set it as no based on that.
3. There may be other tags/options that I might have missed which may be appropriate for this device.

**Comment by Frenzie** [comment]
> I did not try dpi of 167, but it might work, just saw 166 in device specs, either one may be correct.

Please stick to 167 for consistency. :-) (Also 166.6666… rounds up to 167.)

> I'm not sure what hasWifiToggle means, I guessed that it meant having a button to enable/disable WiFi, so I set it as no based on that.

It's about if you can just tell it to toggle the wifi on/off, not a physical toggle iirc.

**Comment by roshavagarga** [comment]
Changed it for consistency, but other devices seem to have it set to 166 instead of 167, so those might need to be changed too?

**Comment by roshavagarga** [comment]
Within the native settings UI, there is a wifi menu, which has an on/off switch, so if that constitutes a Wifi Toggle, then I should probably change that too, right?

File: frontend/device/pocketbook/device.lua [hunk 2]
```diff
@@ -232,6 +232,17 @@ local PocketBook613 = PocketBook:new{
     hasFewKeys = yes,
 }
 
+-- PocketBook 614W Basic
+local PocketBook614W = PocketBook:new{
```
Comment (Frenzie) [inline frontend/device/pocketbook/device.lua:236]:
```suggestion
local PocketBook614W = PocketBook:new{
```

**Comment by roshavagarga** [comment]
Thanks for the merge, taking it that the wifi thing I found isn't a toggle :+1:

**Comment by Frenzie** [comment]
> Changed it for consistency, but other devices seem to have it set to 166 instead of 167, so those might need to be changed too?

If they round to 167, probably. It shouldn't affect anything other than perhaps people with OCD though. ^_^

> Within the native settings UI, there is a wifi menu, which has an on/off switch, so if that constitutes a Wifi Toggle, then I should probably change that too, right?

I didn't really notice this before I merged, but that's more of a thing to determine experimentally. Does the wifi toggle in the KOReader menu work? That's Settings → Network → Wi-Fi connection. You'll probably have to uncomment or remove that `hasWifiToggle` line to test.

**Comment by roshavagarga** [comment]
I'll test it out later, not sure I can enter that part of settings because of the whole hasFewKeys issues.

**Comment by Frenzie** [comment]
You can.

1. Open a book.
2. Press right.
3. Navigate to settings (but I just realized it doesn't rotate, so that might be problematic if you're too far right)

**Comment by roshavagarga** [comment]
If I remember correctly, I was stuck at the furthest to the right, if that one has it, I can test it, if not, no dice :)
Actually, now that I think about it, if both menus rotated and the 'left for leave' thing worked on the bottom menu like on the top one, it would be pretty usable even without remapping keys.

**Comment by Frenzie** [comment]
> If I remember correctly, I was stuck at the furthest to the right, if that one has it, I can test it, if not, no dice :)

You can set the last opened menu in settings.reader.lua.

```
["filemanagermenu_tab_index"] = 4,
-- and probably readermenu_tab_index
```

Removing the line will reset it.

> Actually, now that I think about it, if both menus rotated and the 'left for leave' thing worked on the bottom menu like on the top one, it would be pretty usable even without remapping keys.

Well yes, that's what I proposed, for starters anyway. That's very much with remapping keys though? ;-)

**Pull request merged**